gluten-core/src/main/scala/io/glutenproject/expression/ConverterUtils.scala

```diff
@@ -415,12 +415,14 @@ object ConverterUtils extends Logging {
   final val MIN = "min"
   final val MAX = "max"
   final val STDDEV_SAMP = "stddev_samp"
+  final val COLLECT_LIST = "collect_list"
 
   // Function names used by Substrait plan.
   final val ADD = "add"
   final val SUBTRACT = "subtract"
   final val MULTIPLY = "multiply"
   final val DIVIDE = "divide"
+  final val REMAINDER = "remainder"
```

Comment:
It looks substrait is using `modulus` for remainder function. It would be better to align with substrait, right?